gammapy/detect/test_statistics.py

```diff
@@ -207,10 +330,12 @@ def _ts_value(position, counts, exposure, background, kernel, flux,
     TS : float
         TS value at the given pixel position.
     """
+    from imageutils import extract_array_2d
+
     # Get data slices
-    counts_slice = extract_array(counts, kernel.shape, position).astype(float)
-    background_slice = extract_array(background, kernel.shape, position).astype(float)
-    exposure_slice = extract_array(exposure, kernel.shape, position).astype(float)
+    counts_slice = extract_array_2d(counts, kernel.shape, position).astype(float)
+    background_slice = extract_array_2d(background, kernel.shape, position).astype(float)
+    exposure_slice = extract_array_2d(exposure, kernel.shape, position).astype(float)
```

Comment:
Can you tell me why you switched back from `astropy.nddata.utils.extract_array` to `imageutils.extract_array_2d`? imageutils has been merged with astropy and will not be developed actively any longer. I'm currently working on a PR to improve `astropy.nddata.utils.extract_array` so please let me know if you found any differences between the two that matter for you.